src/sentry/sentry_apps/external_requests/select_requester.py

```diff
@@ -130,14 +133,14 @@ def _format_response(self, resp: Sequence[dict[str, Any]]) -> SelectRequesterRes
 
         for option in resp:
             if not ("value" in option and "label" in option):
-                logger.info(
-                    "select-requester.invalid-response",
-                    extra={
-                        "resposnse": resp,
-                        "error_msg": "Missing `value` or `label` in option data for SelectField",
+                raise SentryAppIntegratorError(
+                    message="Missing `value` or `label` in option data for Select FormField",
+                    webhook_context={
+                        "error_type": "select-requester.missing-fields",
+                        "response": resp,
                     },
+                    status_code=503,
```

Comment:
should this be a 400 if it's during validation?

Reply:
Went with 503, since the client can't fix the issue/ the 3p response was incorrect so the 3p has to fix.